bedrock/teach/templates/teach/smarton/index.html

```diff
@@ -0,0 +1,86 @@
+        <a href="{{ url('teach.smarton.tracking') }}" class="inner">
+          <p class="cta">
+          {% if l10n_has_tag('smarton-surveillance') %}
+            {{ _('Outfox the trackers') }}
+          {% else %}
+            <del>{{ _('Follow me') }}</del>
+          {% endif %}
+          </p>
+        </a>
+        <svg class="shape" viewBox="0 0 255 300" width="100%" height="100%" preserveAspectRatio="none"><path fill="#fff" d="M0,0 v70 c0,0,45,14,130,14 s130-14,130-14 V0 H0 z"/></svg>
+      </li>
+
+      <li class="topic topic-security" id="topic-security" data-path-hide="M0,0 v260 c0,0,60,6,130-5 c80-10,130-5,130-5 V0 H0 z" data-path-show="M0,0 v70 c0,0,45,14,130,14 s130-14,130-14 V0 H0 z">
+        <h2 class="topic-title">
+        {% if LANG.startswith('en') %}
+          {{ _('Cybersecurity') }}
+        {% else %}
+          {{ _('Cyber security') }}
+        {% endif %}
+        </h2>
+        {# L10n: "Buff" here is slang for fit, strong, or tough. #}
+        <p class="topic-intro">{{ _('Look both ways, stay updated, and make sure your passwords are as buff as you are.') }}</p>
+        <a href="{{ url('teach.smarton.security') }}" class="inner">
+          <p class="cta">{{ _('Lock it down') }}</p>
+        </a>
+        <svg class="shape" viewBox="0 0 255 300" width="100%" height="100%" preserveAspectRatio="none"><path fill="#fff" d="M0,0 v70 c0,0,45,14,130,14 s130-14,130-14 V0 H0 z"/></svg>
+      </li>
+
+      <li class="topic topic-surveillance" id="topic-surveillance" data-path-hide="M0,0 v260 c0,0,60,6,130-5 c80-10,130-5,130-5 V0 H0 z" data-path-show="M0,0 v70 c0,0,45,14,130,14 s130-14,130-14 V0 H0 z">
+        <h2 class="topic-title">
+        {% if LANG.startswith('en') %}
+          {{ _('Government Surveillance') }}
+        {% else %}
+          {{ _('Government surveillance') }}
+        {% endif %}
+        </h2>
+        <p class="topic-intro">{{ _('Decide where you think the line should be – and keep others from crossing it.') }}</p>
+        <a href="{{ url('teach.smarton.surveillance') }}" class="inner">
+          <p class="cta">{{ _('Take a stand') }}</p>
+        </a>
+        <svg class="shape" viewBox="0 0 255 300" width="100%" height="100%" preserveAspectRatio="none"><path fill="#fff" d="M0,0 v70 c0,0,45,14,130,14 s130-14,130-14 V0 H0 z"/></svg>
+      </li>
+    </ul>
+  </div>
+</main>
+
+<aside class="section series-description">
+  <div class="content">
+    <h2>{{ _('Smarter by the step') }}</h2>
+    <p>{{ _('Ask yourself what you think, find out what you don’t know, take control of your experience. Then talk it out with your friends.') }}</p>
+
+    {# L10n: <b> elements below are for visual formatting, to distinguish the numeral from the text label #}
+    <ol class="smarton-steps">
+      <li>{{ _('<b>1.</b> Ask') }}</li>
+      <li>{{ _('<b>2.</b> Know') }}</li>
+      <li>{{ _('<b>3.</b> Do') }}</li>
+      <li>{{ _('<b>4.</b> Chat') }}</li>
```

Comment:
This list may be read as "one one" depending on screen readers? I think `list-style-position: inside` can just be used here, but looks like Firefox has an [old bug](https://bugzilla.mozilla.org/show_bug.cgi?id=4522) where a combination of `<ol>` and `display: flex` leads to `0` list items. Hmm....